ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -90,7 +90,8 @@ const EditGasToolTip = ({
 
   return (
     <div className="edit-gas-tooltip__container">
-      {priorityLevel !== PRIORITY_LEVELS.CUSTOM &&
+      {priorityLevel !== PRIORITY_LEVELS.TEN_PERCENT_INCREASED &&
```

Comment:
This condition is a bit hard to parse (I'm surprised that Prettier isn't breaking this up better). What do you think about extracting it to a variable?

Reply:
good point, I just did that